examples/all-clusters-app/cc13x4_26x4/BUILD.gn

```diff
@@ -121,6 +119,13 @@ ti_simplelink_executable("all-clusters-app") {
     defines = [ "CC13XX_26XX_FACTORY_DATA" ]
   }
 
+  if (chip_enable_icd_server) {
+    defines += [
+      "CHIP_DEVICE_CONFIG_ENABLE_SED",
```

Comment:
Fwiw, `CHIP_DEVICE_CONFIG_ENABLE_SED` has been replaced project wide by `CHIP_CONFIG_ENABLE_ICD_SERVER`. you could use that define instead of redefining the old one.